### DISCUSSION TITLE
Questions about the vocal tract

### DISCUSSION THREAD

**Discussion by Xk-w** [schreibfaul1/ESP32-audioI2S#557] [Q&A] [answered]
When debugging with pcm5102a, I found that when using the ESP32 audioI2S library, the left and right channels of MP3 music output are opposite. How should I solve this problem? The library version is v3.0.0 and the MCU used is T-Display S3 (ESP32S3)

**Comment by schreibfaul1** [comment]
actually, that hasn't been noticed over the years. Swapped both channels.

**Reply by Xk-w** [reply, marked as answer]
Just now I tested the latest version of ESP32-audioI2S (Audio.cpp V3.0.6) and found that the problem of the left and right channels of PCM5102A has been successfully fixed, thanks
